hack/tallyvotes.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+import json
+import pprint
+import subprocess
+import sys
+
+if len(sys.argv) < 2:
+    print("Usage tallyvotes.py <PR_URL>")
+    sys.exit(1)
+
+pr_url = sys.argv[1]
+pr_url_parts = pr_url.split('/')
+org_repo = pr_url_parts[3] + "/" + pr_url_parts[4]
+pr_num = pr_url_parts[-1]
+
+maintainers_url = "https://raw.githubusercontent.com/argoproj/argoproj/master/MAINTAINERS.md"
+maintainers = subprocess.check_output("curl --silent "+maintainers_url+" | grep -e '^|.*github.com' | grep Yes | awk -F'[' '{print $2}' | awk -F']' '{print $1}'", shell=True).split()
+
+maintainer_votes = {}
+for manitainer in maintainers:
+    maintainer_votes[manitainer] = 0
+community_votes = {}
+
+pr_meta = json.loads(subprocess.check_output("gh pr view "+pr_num+" -R "+org_repo+" --comments --json comments,reviews", shell=True))
```

Comment:
Does `gh` require additional configurations? Perhaps consider using GitHub's REST API to pull this information (the requests are not frequent here so no authentication is needed either?

Reply:
It does require you to run `gh auth login`, but I suspect the GitHub REST API requires that as well, so even if we switch it would not improve UX? In any case I'll leave this as a future improvement. I don't think I want to spend more time on this since we use this so rarely.

Reply:
Sure. Just for future reference: “ Unauthenticated clients can make 60 requests per hour”. 

https://docs.github.com/en/rest/guides/getting-started-with-the-rest-api#authentication